contents/blog/mobile-session-replay.md

```diff
@@ -0,0 +1,140 @@
+---
+title: How we built mobile replay
+date: 2024-09-18
+author:
+ - ian-vanagas
+ - manoel-aranda-netos
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-engineering-blog.png
+featuredImageType: full
+tags:
+ - Guides
+---
+
+[Session replay](/session-replay) is one of the most powerful tools for understanding user behavior. Web session replay has been a core part of PostHog for a long time now (it was built in our first hackathon), but mobile teams have had to wait longer.
+
+Fortunately, it's finally here on [iOS](/docs/session-replay/ios) and [Android](/docs/session-replay/android) (with [React Native](https://github.com/PostHog/posthog/issues/13269) and [Flutter](https://github.com/PostHog/posthog-flutter/issues/69) coming soon). 
+
+What took so long? Although we had the structure to ingest and playback replays, recording them in mobile apps is much trickier than in web apps. This post goes over why and how we finally managed to overcome them.
+
+## What's so difficult about mobile session replay
+
+Others have complained about [the lack of good mobile replay options](https://medium.com/goodones/15-years-later-there-is-still-no-good-session-replay-for-ios-f8d335999737). Why is that the case?
+
+### 1. Multiple platforms
```

Comment:
Not sure if it makes sense to mention but the Android and iOS SDKs were rewritten before starting session replay for mobile